apps/web/next.config.js

```diff
@@ -219,6 +219,7 @@ const nextConfig = {
     },
   },
   images: {
+    formats: ['image/avif', 'image/webp'],
```

Comment:
PNG fallback happens automatically for browsers that don't support these formats